src/app/events/MessageCreate.ts

```diff
@@ -22,6 +22,9 @@ export default class implements DiscordEvent {
 				await messageCreateOnBountyBoard(message).catch(e => {
 					console.error('ERROR: ', e);
 				});
+			}
+			if (ServiceUtils.isBanklessDAO(message.guild) || message.channel.type === 'DM') {
```

Comment:
We shouldn't need to repeat the same logic check of ServiceUtils.isBanklessDAO(message.guild)